### ISSUE TITLE
if the project a few pages, change the html - page is updated on the first try

### ISSUE THREAD

**Issue by yeger94** [vohof/gulp-livereload#108]

**Comment by cveld** [comment]
I am experiencing that in many cases the signal towards the listening browser is triggered BEFORE the actual file has been changed on disk and/or consumed by the live reload server.
The result being that the page is refreshed, but with the old content. What to do?

**Comment by yeger** [comment]
Hello! I had change reload plugin to browser-synс, and the problem is
solved ... samsa )

2016-08-18 17:43 GMT+05:00 cveld <notifications@github.com>:

**Comment by cveld** [comment]
Indeed, I have changed the stack towards Express/tiny-lr and now everything works fine! There are many tutorials out there to set this up; e.g. http://rhumaric.com/2014/01/livereload-magic-gulp-style/
